src/batch/src/task/test_utils.rs

```diff
@@ -192,46 +192,46 @@ impl<'a> TableBuilder<'a> {
 
     pub async fn run(self) {
         let inserted_rows = self.tuples.len();
-        let create = self.build_create_table_plan();
+        // let create = self.build_create_table_plan();
         let insert = self
             .build_insert_values_plan()
             .expect("failed to create insert plan");
-        assert_eq!(self.runner.run(create).await.unwrap()[0].len(), 0);
+        // assert_eq!(self.runner.run(create).await.unwrap()[0].len(), 0);
         TestRunner::validate_insert_result(
             &self.runner.run(insert).await.unwrap()[0],
             inserted_rows,
         );
     }
 
-    fn build_create_table_plan(&self) -> PlanFragment {
-        let create = CreateTableNode {
-            table_ref_id: None,
-            column_descs: self
-                .col_types
-                .iter()
-                .enumerate()
-                .map(|(i, typ)| ColumnDesc {
-                    column_type: Some(typ.clone()),
-                    column_id: i as i32, // use index as column_id
-                    ..CoreDefault::default()
-                })
-                .collect_vec(),
-            ..Default::default()
-        };
-
-        PlanFragment {
-            root: Some(PlanNode {
-                children: vec![],
-                node_body: Some(NodeBody::CreateTable(create)),
-                identity: "CreateTableExecutor".to_string(),
-            }),
-
-            exchange_info: Some(ExchangeInfo {
-                mode: 0,
-                distribution: None,
-            }),
-        }
-    }
+    // fn build_create_table_plan(&self) -> PlanFragment {
```

Comment:
Is this file still usable? @neverchanje

Reply:
I've confirmed that TestRunner is completely unused. Pls remove them.